security/js-leaks.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JS Global leak test</title>
+    <script src="./leaks.js"></script>
+    <link href='./style.css' rel='stylesheet'></link>
+</head>
+
+<body>
+    <p><a href="../">[Home]</a> ↣ <strong>[JS Global leak Test Page]</strong></p>
+
+    <p>
+        This page checks the global window object for differences versus standard browsers. 
+        This shows protections' footprint that are observable and potentially exploitable by the page.
+    </p>
```

Comment:
let's add one line of instructions how to update those profiles going forward - can be either here or in the README.md (either main or a small readme in this protections' folder)